src/engine_core/src/input/InputManager.hpp

```diff
@@ -19,5 +19,5 @@ class InputManager
 
   private:
     // TODO: Reserve memory for this map???
-    static std::map<KeyCode, KeyData> s_keyDataByCode;
-};
\ No newline at end of file
+    static std::map<KeyCode, KeyData> S_KEY_DATA_BY_CODE;
```

Comment:
Is this naming convention the standard for static properties?
If so, that's cool, just wondering

Reply:
That's the specified one in .clang-tidy, we can change it. :)